app/services/forest_liana/permissions_checker.rb

```diff
@@ -0,0 +1,57 @@
+require 'date'
+
+module ForestLiana
+  class PermissionsChecker
+    @@permissions = nil
+    @@last_retrieve = nil
+
+    def initialize(resource, permission_name)
+      @collection_name = ForestLiana.name_for(resource)
+      @permission_name = permission_name
+
+      @@expiration_in_seconds = !ENV['FOREST_PERMISSIONS_EXPIRATION_IN_SECONDS'].nil? ?
+        ENV['FOREST_PERMISSIONS_EXPIRATION_IN_SECONDS'].to_i :
+        3600
```

Comment:
`@expiration_in_seconds = (ENV['FOREST_PERMISSIONS_EXPIRATION_IN_SECONDS'] || 3600).to_i`

Reply:
Why to instantiate this here? You can do it out of the `initialize` method (at line 7 for example)